integration/apps/sinatra/puma.rb

```diff
@@ -0,0 +1,45 @@
+require 'ddtrace'
+require 'pry'
+
+Datadog.configure do |c|
+  c.profiling.enabled = true
+
+  # # Reconfigure transport to write pprof to file
+  # c.profiling.exporter.transport = Datadog::Profiling::Transport::IO.default(
+  #                                   write: ->(out, data) do
+  #                                     result = nil
+  #                                     filename = "test-#{Process.pid}.pprof"
+  #                                     puts "Writing file #{filename}..."
+  #                                     File.open(filename, "w") { |f| result = f.write(data) }
+  #                                     puts "File #{filename} written!"
+  #                                     result
+  #                                   end
+  #                                 )
```

Comment:
Seeing this in a few places gets me wondering -- have you considered just adding this as a helper somewhere in the gem itself? It seems like something useful even for support -- tell customers to just add a `c.profiling.exporter.export_to_file(...)`  and send us the result.

Reply:
It is a part of a helper (see `demo_env.rb`) I probably didn't finish updating this. It'll be cleaned up though.

Reply:
My suggestion was if it was worth going even further and possibly adding it to the gem, since `demo_env.rb` is something that we'll only have for these tests.

Reply:
I see... interesting suggestion. I think this could be really nice actually; please feel free to add this to the backlog @ivoanjo !

Reply:
Done: https://datadoghq.atlassian.net/browse/PROF-2767